frontend/.env.sample

```diff
@@ -1 +1 @@
-VITE_API_URL=http://127.0.0.1:5173
\ No newline at end of file
+VITE_API_URL=http://127.0.0.1:8080
```

Comment:
Add empty line

Reply:
And why did you change the port?

Reply:
> And why did you change the port?

To connect frontend to our backend, or have i misunderstood something?

Reply:
The default port Vue app is running is 5173.
You have defined CORS_ALLOWED_ORIGINS in such a way
```
CORS_ALLOWED_ORIGINS = [
    "http://127.0.0.1:5173",
    "http://localhost:5173",
]
```

This means that CORS will allow only 5173 port to connect to your backend. Setting the port to 8080 means that your frontend app will run at 8080 port and won't be able to connect to the backend

Reply:
@olena-hul
So in this case we should leave default value VITE_API_URL=http://127.0.0.1:5173?

Reply:
Either leave default or change CORS_ALLOWED_ORIGINS if you're running frontend on port 8080. How was your app working using different ports in VITE_API_URL and CORS_ALLOWED_ORIGINS?)

Reply:
Thanks for your answer )